integration/kubernetes/helm-chart/alluxio/templates/proxy/service.yaml

```diff
@@ -0,0 +1,43 @@
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+{{- if not (eq .Values.proxy.enabled false) -}}
+
+{{ $proxyCount := int .Values.proxy.count }}
+{{- $release := .Release }}
+{{- $name := include "alluxio.name" . }}
+{{- $fullName := include "alluxio.fullname" . }}
+{{- $chart := include "alluxio.chart" . }}
+{{- range $i := until $proxyCount }}
+  {{- $proxyName := printf "proxy-%v" $i }}
+apiVersion: v1
+kind: Service
```

Comment:
I know the reason for defining multiple `Service`s is for the `StatefulSet` but I'm wondering if the proxy pods need to be stateful or not. I'll have to look into whether or not clients care if they reach the _same_ proxy each time or not.